skopt/tests/test_gp_opt.py

```diff
@@ -71,3 +73,17 @@ def test_gpr_default():
     """Smoke test that gp_minimize does not fail for default values."""
     gp_minimize(branin, ((-5.0, 10.0), (0.0, 15.0)), n_random_starts=1,
                 n_calls=2)
+
+
+@pytest.mark.fast_test
+def test_use_given_estimator():
+    """ Check that gp_minimize use the given estimator. """
```

Comment:
I'd change this to `# Test that gp_minimize does not use default estimator if one is passed in explicitly.`